### DISCUSSION TITLE
Possible to use Tauri instead of electron to build a desktop IDE?

### DISCUSSION THREAD

**Discussion by B-Teague** [eclipse-theia/theia#9888] [General]
Is it possible to build a desktop IDE using Tauri instead of electron?

**Comment by vince-fugnitto** [comment]
@B-Teague thank you for the discussion, I believe others in the past have tried (from the `tauri` team themselves):

- https://github.com/eclipse-theia/theia/issues/4541#issuecomment-640484733
- https://github.com/tauri-apps/tauri-theia
